tests/unit/test_charm.py

```diff
@@ -579,6 +600,54 @@ def test_team_map():
     )
 
 
+def test_swift_config(
+    harness: ops.testing.Harness,
+    setup_replica_consensus: typing.Callable[[], dict],
+):
+    """
+    arrange: after peer relation established and database ready
+    act: update legacy version of the wp_plugin_openstack-objectstorage_config configuration
+    assert: parsed swift configuration should update all legacy fields.
+    """
+    harness.set_can_connect(harness.model.unit.containers["wordpress"], True)
+    setup_replica_consensus()
+    harness.update_config(
+        {
+            "wp_plugin_openstack-objectstorage_config": json.dumps(
+                {
+                    "auth-url": "http://10.126.72.107/identity/v3",
```

Comment:
Where does this IP come from?

Reply:
It's just a random IP address from my local devstack server, the IP address here does not matter.

Reply:
I changed the IP address to `swift.test` to be self-explanatory.